project.md

```diff
@@ -5,47 +5,79 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project will serve people who create digital monthly calendars and want to make them more easily. My motivation to take on this project stems from the idea of professors and students using my DSL together. I imagine professors giving a text file that contains all important due dates for their class to their students written in my language. Since the text file will be written in my language, it can then be able to be passed through an application that reads the file and adds the items to a calendar. This would allow students to easily create a visual representation for all assignments that they have due over the course of the semester, instead of having to create their own manually or referring to a calendar created by the professor on the class website or syllabus.  
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+My idea will meet the needs of students and anyone who creates a digital calendar on their computer. As a student, at the beginning of each semester, I go through the syllabuses for each of my classes and create a calendar that contains due dates for all of my upcoming assignments. Since I learn and remember things visually, I like to create a monthly where I can see everything at once. Thus far, I have manually created a calendar in excel for each semester I have been in college. This process takes upwards of two hours and involves setting the dates for each school week, scraping each syllabus for assignment due dates, and determining what is worthy of putting on my calendar (some professors make this easier/more clear than others as they clearly outline what is due on their syllabus), and color coding assignments by class. 
+  
+If I can help myself, and others who face this problem, I would be able to create a long-term calendar by first initializing a desired length for their calendar and then by passing in a file with items they want to add. What makes my DSL unique and helpful is that it will allow users to easily add details when inputting their items instead of altering them manually afterward. These include:
+  - Reoccurance: The user can specify if they want the item to reoccur by week, month, etc.
+  - Color: the user can easily manipulate the color of the item by specifying it in the input
+  - Bold/underline/italicizate
```

Comment:
Some things I noticed in my due dates calendar in iCal form (that I manually put in one-by-one on Google, which you pointed out its pains perfectly in this project description!) that may help you start your syntax design around these themes:

**reoccurance**  - there is an “RRULE” key that has configurations/flags under it for reoccurance.
**color** - this may also be an indirect way (depending on end user) to say group these x events together. This can be done on google calendar by making separate icals and then format that entire set to a specific color (instead of one by one).

I agree having an intermediate DSL is a really useful way to approach this problem since you are juggling between multiple formats (nice syntax for niche functions to events, then events to desired, existing way of organizing these events).